pages.pt_PT/common/yt-dlp.md

```diff
@@ -0,0 +1,37 @@
+# yt-dlp
+
+> Um fork do youtube-dl com funcionalidades e correções adicionais.
+> Descarrega vídeos do YouTube e de outros websites.
+> Mais informações: <https://github.com/yt-dlp/yt-dlp>.
+
+- Descarregar um vídeo ou playlist (com as predefinições do comando abaixo):
+
+`yt-dlp "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Descarregar um vídeo com um formato definido. Neste caso, fundindo o melhor formato de vídeo com o melhor formato de áudio (Predefinição):
+
+`yt-dlp --format "{{bv*+ba/b}}" "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Extrair áudio de vídeos (necessário ffmpeg ou ffprobe):
+
+`yt-dlp --extract-audio "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Especificar o formato de áudio extraído (best(predefinição), acc, flac, mp3, m4a, opus, vorbis, wav, alac):
```

Comment:
Não é o propósito do tldr-pages ter a listagem exaustiva de todas as opções de um comando. Acho que ficaria mais natural dizer simplesmente isto:

```suggestion
- Especificar o formato de áudio extraído (a predefinição é "best"):
```

Reply:
> Compreendo, também faria sentido mudar a versão inglesa, correto?

Exato :)

Reply:
És a favor de ter a discussão e mudança das docs inglesas neste PR ou achas que faz mais sentido abrir um diferente? 🙂

Reply:
Applied in [`7b15476`](https://github.com/tldr-pages/tldr/pull/8936/commits/7b15476ca39f71c5417417a84fcc5f532ff393f3).